documentation/release_5.0.htm

```diff
@@ -340,12 +340,24 @@ <h3>Backward incompatibities</h3>
       You will get a run-time exception if you forgot to do this.
     </li>
     <li>
-        Rotational changes to STIR's interface to ROOT files, <a href=https://github.com/UCL/STIR/pull/1012>PR #1012</a>. In previous versions (roughly) half a bucket,
-        <code>(module_repeater_y * submodule_repeater_y * crystal_repeater_y) /2 - 1)</code>
-        in the <tt>Cylindrical</tt> case, was subtracted from the crystalIDs, in an attempt to rotate
-        the scanner. However, this was incompatible with the STIR code to find normalisation factors,
-        and could be surprising for odd numbers etc.
-        Rotations should use the new view offset support described above.
+        Rotational changes to STIR's interface to ROOT files, <a href=https://github.com/UCL/STIR/pull/1012>PR #1012</a>
+        and <a href=https://github.com/UCL/STIR/pull/1016>PR #1016</a>.
+        <ul>
+            <li> In previous versions (roughly) half a bucket,
+            <code>(module_repeater_y * submodule_repeater_y * crystal_repeater_y) /2 - 1)</code>
+            in the <tt>Cylindrical</tt> case, was subtracted from the crystalIDs, in an attempt to rotate
+            the scanner. However, this was incompatible with the STIR code to find normalisation factors,
+            and could be surprising for odd numbers etc.</li>
+            <li>Similarly, the detector numbers of each ROOT event were offset by <code>quarter_of_detectors</code>.
+            This was because of the assumption that GATE the 0th crystal is on the on the x axis (STIR's is on y).
+            However, the usage of <code>quarter_of_detectors</code> requires the number of scanner
+            crystals/detectors to be factorisable by 4, which may not always be the case.
+            This addition of <code>quarter_of_detectors</code> has been removed and GATE scanners are expected
+            to be rotated by -90 degrees so the 0th crystal is on the on the y axis. </li>
+            <li>The <code>offset (num of detectors)</code> parameter has been removed from ROOT header files and
+            and all underlying methodology removed.</li>
+        </ul>
+        Rotations should use the new <code>View offset (degrees)</code> support described above.
```

Comment:
```suggestion
        Pre STIR 5..0, STIR renumbered crystals obtained from the ROOT file to rotate the reconstructed image.
        This was because GATE macros that we used placed the first bucket (i.e. Gate module) on the <i>right</i>
        of the scanner, while STIR by default places the 0th crystal is on the <i>top</i>.
        This was done by adding various offsets to the crystal ID. However, this was incompatible with the 
        STIR code to find normalisation factors, which assumes that a bucket starts at crystal 0.
        Moreover, the offsets used integer division which could be surprising for odd numbers.
        Specific offsets that have been removed:
        <ul>
            <li> (roughly) half a bucket,
            <code>(module_repeater_y * submodule_repeater_y * crystal_repeater_y) /2 - 1)</code>
            in the <tt>Cylindrical</tt> case, was subtracted from the crystalIDs</li>
            <li>(roughly) 90 degrees, <code>quarter_of_detectors=get_num_detectors_per_ring()/4</code>.</li>
            <li>An optional<code>offset (num of detectors)</code> parameter in the `.hroot` and
            and all underlying methodology has been removed.</li>
        </ul>
        Rotations should use the new <code>View offset (degrees)</code> support described above.
        See also the STIR User's guide.
```